src/hooks/useDeviceStateTracker.ts

```diff
@@ -0,0 +1,67 @@
+import type { Ref } from '@muban/muban';
+import { onMounted, onUnmounted, ref } from '@muban/muban';
+
+import DeviceStateTracker, { DeviceStateEvent } from 'seng-device-state-tracker';
+import type IDeviceStateData from 'seng-device-state-tracker/lib/IDeviceStateData';
+
+import sharedVariables from '../config/shared-variables.json';
+
+const cleanMediaQueries = Object.keys(sharedVariables.mediaQueries).reduce<{
+  [key: string]: string;
+}>((result, key: string) => {
+  // eslint-disable-next-line no-param-reassign
+  result[key] = sharedVariables.mediaQueries[key].replace(/'/g, '');
+  return result;
+}, {});
```

Comment:
```suggestion
}>((result, key: string) =>({ ...result, [key]: mediaQueries.mediaQueries[key].replace(/'/g, '') }), {});
```
If you change it to this, you don't need to disable the linter